### DISCUSSION TITLE
Data Feminism: The Numbers Don't Speak for Themselves

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-AY24#462] [Reading Responses]
Read and respond to the ["Situating Data on the World Wide Web" section of Data Feminism](https://data-feminism.mitpress.mit.edu/pub/czq9dfs5#situating-data-on-the-wild-wild-web)

Questions to get you started:

1. In the reading, they say "Ironically, some of the most admirable aims and actions of the open data movement have worked against the ethical urgency of providing context, however inadvertently." Based on your experiences and the reading, what are some situations in Earth Data Science where lack of context could be misleading?
2. What are some things you can do as a Data Scientist to provide more context for your analysis? What are some of the obstacles  you might face?
3. What are some things you can do as a data provider to help scientists keep data in context?

**Comment by BioNCphilic** [comment]
I've been trying to get through the [Transform To Open Science modules](https://github.com/nasa/Transform-to-Open-Science) which address metadata as very important. One example provided is "Are There Any Harms to Open Data?
Open data has been demonstrated to further marginalize or exploit small-scale and community driven initiatives, such as in [the case](https://thebulletin.org/2022/05/how-a-dispute-over-sharing-coronavirus-genomes-is-threatening-a-vital-tool-for-tracking-variants/) of African researchers neither receiving due credit nor compensation for their genome sequencing during the COVID-19 pandemic.

NASA and other federal agencies are now requiring Data Management Plans when submitting proposals for funding. This includes: 

A description of data types, volume, formats, and (where relevant) standards;
A description of the schedule for data archiving and sharing;
A description of the intended repositories for archived data, including mechanisms for public access and distribution;
A discussion of how the plan enables long-term preservation of data;
A discussion of roles and responsibilities of team members in accomplishing the OSDMP. If funds are required for data management activities, these should be covered in the normal budget and budget justification sections of the proposal.

**Comment by annaposlednik** [comment]
Lack of context in earth data science can be especially misleading when connecting back to human populations, for example in analyses that look at anthropogenic drivers of climate change and the disproportionate effects of climate change on various communities. I do find that the lack of easily understandable metadata in most online datasets is a real challenge in my own data science work, and there should be a more streamlined format for metadata of all published datasets. As a data provider, I can be very clear and thorough about how my data are collected and include important caveats of missing/incomplete data. An obstacle to this however might be not being aware of biases in my own data collection methods, as well as any privacy concerns when dealing with sensitive data that could unintentionally harm communities.

**Reply by BioNCphilic** [reply]
Good point - biases may not be apparent to the data provider.

**Comment by allenmoench** [comment]
1) Open data creates a vast pool of supposedly “raw” data and numbers which can be freely processed and utilized by anyone from trained professionals to the public at large. The risk is that people will fail to understand the context of the data, and subsequently misuse it, or even intentionally misrepresent the numbers to support a desired narrative.

There are many places in Earth Data Science where we could potentially run into this. I’ll use climate science as an example. Climate science involves sampling and processing vast quantities of data. A possible source of error could come from failing to understand sampling methods. Another one could come from confirmation bias, for example the assumption that because climate is known to be changing, that we should expect temperatures to increase. In many places we may see decreases instead, and as good scientists we shouldn’t be mislead by what we expect to see. Climate data could also be edited and used in ways that support a particular narrative about whether or not the climate is changing.

2) When using data, make sure to do your due diligence and research deeply enough to understand the context of the data so that it isn’t just a collection of numbers. I like Heather Krause’s set of five questions mentioned in the article: Where did it come from, who collected it, when, how was it collected, and why. If I began each analysis by asking and answering these questions, I think I would feel confident that I had at least a basic understanding of the data - a sort of “heads-up” approach that hopefully should catch any major problems. Of course, more extensive analyses might require more extensive methods of context research.

3) Documentation seems to me to be the best way to put data properly in context. When creating data, make sure the methods of data creation are clearly explained, available, and comprehensible even to an untrained reader. If collecting data directly, for example by collecting samples in the field or taking a questionnaire, do your best to collect data that are consistent and representative, as there are lots of things that can skew or confound the results and these need to be diligently avoided. Try to avoid biases and get the “objective” picture wherever possible (easier said than done).

**Comment by gpb3037** [comment]
1. Some situations in Earth Data Science where lack of context could be misleading include having different precision between sources interpreted as being similar.  I have anecdotal knowledge of the Jeddah Convention which is a Regional Convention for the Conservation of the Red Sea and Gulf of Aden Environment that was signed in 1982 by all Red Sea coastal nations. The convention's goals include protecting the marine environment from pollution and managing marine resources.   My understanding is that contributions to funding the efforts were determined by the length of coastline of each nation, but that those who had more precise data in effect had longer coastlines which considered more undulations that those that had lower resolution - this mismatch in a way penalized those with more precise data and there needed to be a way to generalize or de-resolve the data in a similar way to the lowest common precision.

2.  A way to provide more context may be some method of blending qualitative and quantitative observations when possible. "Five by five" is a radio term used to describe a strong signal that is also clear and loud.  As part of an analysis one might have some factor attributed to data that adds a quantitative rating to qualitative observation and vice versa.  An obstacle to this may be coming up with a consistent scale between observers.

3. As a data provider it is important not to infer that a source is the best based on current use,  or not consider a that source may have possible restriction by omission or outside influence.   What might be deemed unimportant in terms of the description of sources and methods in collection and analysis today may not always be the case as new techniques evolve.  A typical census has collected a basic set of demographic data, but now has added small samples of more detailed information which can be modeled to reflect changes in the whole, as long as a proper regimen of statistical sampling is followed and not skewed by convenience.

**Reply by BioNCphilic** [reply]
Interesting example from the Red Sea. Thanks.

**Comment by lauren-pease** [comment]
Earth Data Science must demonstrate an examination of the given context as well as express the contextual limitations of the data and methods of data collection and analysis in order to explain phenomena with some degree of validity. Otherwise, the science produced will remain at a consequential distance from ground truth. The fallacy of raw data removes responsibility from the data scientist to correctly name and interrogate the data. Data collection is itself a form of storytelling: what is under observation and why? What are the cultural values of the data? For whom does the data exist? What data is not collected and unseen by data scientists? Does this obfuscation perpetuate what Black feminist intellectual Patricia Hill Collins describes as subjugated knowledge which effaces both knowledge and people? 

Approaches supporting contextual examination may be in the form of a data biography or data user guide. Data scientist Heather Krause created the data biography framework which asks the where, who, when, how, and why questions of data collection. This can be easily adapted into existing metadata schemas. Data user guides offer extensive dataset explainers which require the development of deeper understanding such as data purposes and limitations. These guides are probably best suited to be additions within a publication.

Furthermore, during analysis data scientists should exercise a greater reliance on theory rather than intuition to counter a biased interpretation of a dataset. Human-Earth interactions and corresponding data may be interpreted by feminist theory such as ecofeminism (defined by Françoise d’Eaubonne in 1974) which emphasizes relations with Earth are likely indicators of the treatment of women in society under patriarchal structures or perhaps a branch within the theory known as subsistence feminism (championed in _Ecofeminism_ by activist and coauthor Vandana Shiva in 1993) which advocates particularly for the concerns of non-Western women to meet basic needs such as food and shelter in addition to societal needs Western women generally call for like equality and liberation (Thomas, 2022). Even still, this theory and its branches are called into question for erasing an intersectional perspective (theory coined by Kimberlé Williams Crenshaw in 1989) that centers Black, Latine, Indigenous, and Asian women as well as nonbinary, trans, and queer people and other underrepresented folks. Educator and eco-communicator Leah Thomas is a proponent of the approach of intersectional environmentalism (2022) which draws on an intersectional adaptation of ecofeminism and extends it to encompass additional intersections of identity including religion, race, sexuality, age, ability, and class. The approach “embraces all cultural contributors to sustainability and environmentalism” in an effort to provide a comprehensive explanation of environmental history. By leveraging theory in Earth Data Science, the data collected, studied, and communicated will more effectively serve both data scientists and people affected by the data narratives. 

Reference

Thomas, L. (2022). _The intersectional environmentalist: How to dismantle systems of oppression to protect people + planet_. Little, Brown and Company.

**Reply by BioNCphilic** [reply]
Data Biographies are a really interesting idea and are being tested for educational purposes in high school classes, but I can see them as very useful in publications too. Thanks for the reference book!

**Comment by npwelsh** [comment]
1. When thinking about lack of context in data in day to day conversation, my mind goes to renewable and fossil fuel power and energy number are often skewed to show intrenched interest, usually on the side of fossil fuel.  For example, in winter storm Uri in Texas, wind was thought as the culprit for the power grid becoming unstable and shutting off power for millions of people, as people talked with wind having a low capacity factor, or total energy output, than natural gas.  However, the reality was is that natural gas, even with its higher capacity factor a able to be dispatched at will, was the main culprit during winter storm Uri, as 58% of natural gas plants experienced outage at some point, as both natural gas supply and equipment was not equipped for the weather, compared to 27% of wind generation (ferc.gov). 
2. Thinks I have tried to do to include more context are provided a writeup about a data set, include a read me about how to use the data properly, and cite my sources or collection methods.  While these are all good, they do take more time, both to create and for others to read and use.  This might lead to slow down in data generating and synthesizing, which may or may not be a bad thing.  Another obstacle might be sources with poor context, or even collection with poor context.  That might throw off my own write up of the data. 
3.  As a data provider, I can clearly document all the steps I took to collect that data.  I might also include examples where it is appropriate to use the data and where it is not.  I can also highlight short coming or blind spots in my data, where I highlight what might be missing or my own biases have caused me to improperly sample something or not sample it.

**Comment by hanried** [comment]
When completing data analyses, it is important that I work to build the habit of doing my best to cite and explain the data I’m using. It’s been a part of our portfolio posts so far and it sort of feels like a tedious task and an afterthought since I feel so focused on getting the code to work. However, in light of the reading and trying to write meaningful explanations for my own code, I see how important it is to explain what data we’re using and where it is coming from. I need to make sure that I have a good understanding of what data I’m working with so that I can help viewers understand my analysis better. I do think providing context for our data can be tricky if the metadata or the data source doesn’t provide clear context to start with.

**Reply by BioNCphilic** [reply]
Yes - this is especially true when trying to go back to something you did in class days earlier. Good example!

**Comment by ZimmZone** [comment]
I felt that the use of data feminism in the context of this post served more to confuse than to clarify. Instead of characterizing large out-of-context and aggrandized data sets as Big Dick Data (which is certainly an evocative name) I would have stuck with highlighting that the researchers using this data out-of-content as failing to do due diligence in their research. I felt that it unnecessarily and unfairly equated masculine data as 'bad research' and feminine data as 'good research'. I feel that creating a dichotomy between masculine (without context) and feminine (within context) was out of step with the other 'feminisims' in the fields of the social and political sciences. For instance, for me feminist data would be data that acknowledges and accounts for specifically minority voices and context  - and not contextualization in general.

However, this was not one of the questions we were asked to address. 

In large data sets, especially those that are anonymized, I would think that most data would have a tendency to be misleading. Using the example of exit polling. If 'exit polls' indicate a specific candidate will win it is ESSENTIAL to know where those polls were taken and if they represent the voting demographics of the United States. If the exit polls are only taken is specific areas, and the poltical leanings of these areas are not identified (e.g. rural Kansas and downtown San Francisco) then they exit polls will not be an accurate representation of the election outcome. 

Perhaps more inadvertent is an acknowledgement of the digital divide. Many open data sets are created by opt-in users, including the date we are using for this assignment. This means that all individuals contributing to these data sets are computer literate, understand how to opt in and have the intellectual interest to do so, and have access to the internet. This is actually a quite limited demographic as there are still many places that lack reliable access or universal computer literacy. These places may be awash with the species you are interested in  but have zero reports due to technical limitations. 

I agree with the article that the best way to give context is to seek out data sets that are already clear in their sources and limitations. If you have to use other data sets, conduct the due diligence to answer as many of the who, what, and why questions as possible and attempt to add context of your own.  If context is limited, do not base your entire analysis upon the data set but seek additional sources. 

One way I can help keep data in context is to only knowingly contribute to data sets that gather sufficient contextual information. Second, I can limit my use of sub-par data sets or take extra steps to ensure that I add context in addition to that provided by the data itself.

**Reply by BioNCphilic** [reply]
"I  felt that the use of data feminism in the context of this post served more to confuse than to clarify." Thanks for mentioning the double take on data context. I agree there was more to unpack than the issue we are discussing - providing enough context for data to be used appropriately and fairly.

**Comment by ZacharyCJDecker** [comment]
_In the reading, they say "Ironically, some of the most admirable aims and actions of the open data movement have worked against the ethical urgency of providing context, however inadvertently." Based on your experiences and the reading, what are some situations in Earth Data Science where lack of context could be misleading?_

Earth Data Science typically relies a lot on observational datasets. How that data was collected and processed can alter the results significantly. My experience mainly relies on producing earth datasets in very specific and unique situations: atmospheric datasets in a limited area and time period. Before these data are published online there is a significant amount of "data reduction". We publish most of those details with the data, but if it were not then using that data would be nearly impossible to understand the data. These data are also useless if they are not considered in conjunction with additional datasets. For example, one could pull data on CO2, but without corresponding data on location, time, wind direction, etc. these data could not be interpreted. Even more, meteorology plays a huge role in the interpretation, but that is not a dataset that can be downloaded. It requires expert understanding to account for meteorology 


_What are some things you can do as a Data Scientist to provide more context for your analysis? What are some of the obstacles you might face?_
Providing metadata is essential. Details on how the data are reduced should be a requirement. In many cases peer-reviewed articles are published this the data reduction methods to make this procedure referenceable. However, sometimes new instrumentation does not have a clear data reduction process and certainly not a published process. These reduction processes can then change over time as better understanding it gathered. Updating that meta data is important in those cases. 

_What are some things you can do as a data provider to help scientists keep data in context?_
Providing metadata and examples of the use and limitations of the data would be important to keep the data in context.

**Comment by byandell** [comment]
1. _In the reading, they say "Ironically, some of the most admirable aims and actions of the open data movement have worked against the ethical urgency of providing context, however inadvertently." Based on your experiences and the reading, what are some situations in Earth Data Science where lack of context could be misleading?_
I have been watching (on Netflix) the movie [Coded Bias](https://www.codedbias.com/), which focuses on five powerful women--Joy Buolamwini, Cathy O'Neil, Meredith Broussard, Silkie Carlo and Virgina Eubanks--delving into the machine bias of facial recognition, and how this sits in the larger context of our emerging digital world.
 
2. _What are some things you can do as a Data Scientist to provide more context for your analysis? What are some of the obstacles you might face?_
Document, document, document. Build provenance about the data, code, and project area as I go.
This is time consuming, and it is hard to go back later and fill in the blanks.

3. _What are some things you can do as a data provider to help scientists keep data in context?_
The data I proved are mostly code and repos. Cross-referencing material up and down the infotree (metadata across projects) is important to keep things in context. I also build some of this into my homepage and blog.

**Comment by VictoriAragon** [comment]
1. As a scientist, I've recognized that context matters the most when studying and overall trying to explain climate change and weather patterns, extreme events and outliers in the data. It's easy to pass an outlier or extreme weather event as climate change and thereby, ignoring your own personal bias. 

Especially when it comes to climate change, its important to not denote the subject to bias due to sensitivity of listeners on the matter. This is where you can understand and explain the context and then give a non-bias analysis of data that you observed. 

2. Some things to give data more context is to go deep into the source, metadata, and dated trends. Then your data becomes factual and doesnt leave the door open to criticism where personal bias is open to be used against you. 

3. As a scientist, I follow scientific code of ethics and data that I collect whether physical or online will be provided in the context that it was collected using a code of ethics. Any data I am unsure of, I will not provide online to the public because my name is attached to that data.

**Comment by atticusbaker** [comment]
Context is super important when using any data. In my experience, for instance with temperature data over time, it is great if the data are readily accessible, but without the important context of where it was collected and other important local events that happened near the data collection device, then the data is almost useless.

To provide context, any data scientist must make sure to write down any and all information relating to your data. Make sure that metadata is clear, concise, and filled to the brim with extra data about how the data was collected, stored, used, etc. It only makes it easier for the next user of the data.

To keep data in context, you must be sure to include all the information you can about it, but also make sure that you and others are using data ethically, specifically avoiding data that could have been collected in bad faith or altered in some way.

**Comment by brglea** [comment]
* Based on my experiences (in the class so far - I don't work in this field) and the reading, some situations in Earth Data Science where the lack of context could be misleading, may be dealing with crowdsourced data. To my knowledge, crowdsourced data doesn't actually say *who* collected the data (which may be related to data privacy). As a possible scenario, if most users that upload data to iNaturalist are men, or are mostly white, educated, I think that also tells a different story that isn't told and may influence analysis of the data. 

* Some things I can do as a data scientist to provide more context for my analysis are never take the numbers in a dataset at face value due to the imbalances of power in data setting. I need to interrogate the "context, limitations, and validity of the data" I am using and ask myself what is missing.  I need to ensure that the data's "situatedness" is taken into account as I am building upon whatever dataset that might be, with my analysis. It's kind of like needing a foundation to build a house; without having sound structural support upon which to build, a house would fall. I think most things in life, fail in one way or another without proper context and being situated.
Some struggles I might face to provide more context for my analysis are a lack of context or not enough context coming from the dataset I am using.  Also, a lack of time or resources to provide context could also be a struggle, however I couldn't in good conscious publish analysis on something without context, this seems unethical to me. Any struggles or issues I face during analysis need to be called out in limitations to the analysis.

* Some things I can do as a data provider to help scientists keep data in context are:

1. When I am gathering data or creating a dataset I need to understand the mindset that I am cooking data and call myself out on what is missing from my dataset (what was left out) and what that tells someone about the dataset. 
2. Explicitly say how the dataset was intended to be used (or how it should not be used) with clear variable/table/header names, even a dictionary of definitions for certain words.
3. Incorporate culture, context, and nuance to gathering data and creating datasets.
4. Encourage data users to interrogate a dataset I create, but also provide my own interrogation as a starting point - "ask questions about the social, cultural, historical, institutional, and material conditions under which that knowledge was produced, as well as about the identities of the people who created it.”  There's nothing I love more than being self-critical. 
5. Limitations are key and thoroughly explaining what those are and any associated ethical obligations.

 

Aside from answering these questions I wanted to say that I LOVED this reading (bought this book immediately). I went to a women's college, one of the seven sisters, and feminist theory is entrenched into my being and everything I do. If I could, I would take a whole class or certificate in Data Feminism or Data Intersectionality because it's empowering and awesome. 

The context problem is data science I think speaks to how this field is heavily dominated by men and has been since it's creation. I am not shocked of the 'issue of context' because of this. 

After reading this chapter, it made me think of some great authors - Kimberle Crenshaw who coined 'intersectional feminism' and she has great TED talks that are very moving [The Urgency of Intersectionality](https://www.ted.com/talks/kimberle_crenshaw_the_urgency_of_intersectionality?subtitle=en) . As well as Chimamanda Ngozi Adichie - [Danger of a Single Story](https://www.youtube.com/watch?v=D9Ihs241zeg) and [We Should All be Feminists](https://www.youtube.com/watch?v=S6ufvYWTqQ0) .

**Comment by amarangunic** [comment]
I'll come back to something that I always find disturbing about the lack of context in the data on the web related to the COVID data reports, as in many cases the numbers shared did not represent the full reality. The numbers presented often did not take into account factors such as demographics, accessibility of evidence, reporting delays, and biases in health systems. This led to misinterpretations and misinformed decisions. As in the text, the need to place data in context to avoid erroneous or biased conclusions is emphasized.
T
he relationship I see between the Data Feminism text and the COVID data reports lies in the lack of adequate context to correctly interpret the data. During the pandemic, numbers of infections, hospitalizations, and deaths were often reported without considering critical factors such as testing capacity, regional variations in access to care, or delays in data collection. This led to an incomplete or inaccurate understanding of the situation. Similarly, Data Feminism stresses that data, when not placed in their social or structural context, can perpetuate inequalities or lead to misinterpretations.

For example, COVID data could have been manipulated or misinterpreted based on unequal access to testing in different communities, which could have obscured or exaggerated the impact of the disease on certain populations. Indeed, without information about who was being tested, how the data were collected, and under what conditions they were shared, there was a risk of making decisions based on incomplete information.
Data Feminism also points out how data that lacks situated understanding can perpetuate inequalities, and this was also evident during the pandemic, as marginalized communities were often more impacted by the virus, but data did not always fully reflect this reality due to biased or unequal systems.

This critical approach to data highlights the importance of assessing the systems and contexts in which data are produced and reported, as it is not sufficient to simply present numbers without an understanding of the factors that generate them.
It is true that during the COVID pandemic, the lack of detailed information on the location of cases was a major challenge. Although the use of Big Data made it possible to quickly analyze large volumes of data, often these data did not include critical details about the local situation. This made it difficult to make accurate decisions at the regional or community level, which could have helped to target resources more effectively. The rapid spread of the virus also complicated data processing and analysis in real time.

The lack of local data also created inequities, as some areas may have needed more support but did not receive adequate attention due to low visibility in national or global reporting. The use of Big Data without sufficient situated context (as argued by Data Feminism) can lead to interpretations that ignore the local, social, and economic conditions that influence the spread of diseases such as COVID-19.

To help keep the data in a secure context, some forms such as complete metadata, sources, date, etc. Provide clear documentation of where and how the data was collected and processed, include social and geographic context information, with frequent updates. In addition to explanations of biases or errors, with transparency in data collection. This and more details could ensure that the data are interpreted correctly, which is the big problem, the interpretation of the data.
